plugins/SliceInfoPlugin/SliceInfo.py

```diff
@@ -119,6 +119,7 @@ def _onWriteStarted(self, output_device):
             }
             for container in global_container_stack.getContainers():
                 container_id = container.getId()
+                container_serialized = ""
```

Comment:
I don't see why this is necessary? The `if` clause below can only be reached if the `try` did not throw an exception, and if the `try` did not throw an exception, `container_serialized` will always be overwritten by the result of `container.serialize()`. Q.E.D.?